### PR TITLE
Getters for graph conversion

### PR DESCRIPTION
**Pull request by iliailia**
Some helpers are added to convert from plaintext to secret shared graphs. `from_another` is updated to support closures instead of functions.

### CODE REVIEW

File: src/graph_store/graph_mem.rs [hunk 1]
```diff
@@ -18,19 +18,31 @@ impl<V: VectorStore> GraphMem<V> {
...
+    pub fn get_layers(&self) -> Vec<Layer<V>> {
+        self.layers.clone()
+    }
 }
 
 // Plain converter for a Graph structure that has different distance ref and vector ref types.
 // WARNING: distance metric is assumed to stay the same; thus, conversion doesn't change the graph structure.
 // Needed when switching from a PlaintextStore to a secret shared VectorStore.
 impl<V: VectorStore> GraphMem<V> {
-    pub fn from_another<U>(
```
Comment (rdragos) [inline src/graph_store/graph_mem.rs:27]: is there a way we can add a test for this functionality?

**Review by rdragos — APPROVED** [review]
great work! Just a small nit re testing

File: src/graph_store/graph_mem.rs (hunk 1 above)
Comment (iliailia) [reply src/graph_store/graph_mem.rs:27]: I can quickly add a naive one using `LazyMemoryStore`, where this function does nothing. It isn't hard to add one more example struct satisfying VectorStore and then do more involved tests, but some non-trivial amount of boilerplate should be written.

**Review by rdragos — APPROVED** [review]
Nice work! :rocket:

**Pull request merged**